pkgs/by-name/mq/MQTT-Explorer/package.nix

```diff
@@ -0,0 +1,181 @@
+{
+  lib,
+  stdenv,
+  electron,
+  yarn,
+  fixup-yarn-lock,
+  fetchFromGitHub,
+  fetchYarnDeps,
+  nodejs,
+  typescript,
+  makeWrapper,
+  makeDesktopItem,
+  copyDesktopItems,
+}:
+let
+  electronDist = electron + (if stdenv.isDarwin then "/Applications" else "/libexec/electron");
+in
+# NOTE MQTT-Explorer has 3 yarn subpackages and uses relative links
+# between them, which makes it hard to package them via 3 `mkYarnPackage`
+# since the resulting `node_modules` directories don't have the same structure
+# as if they were installed directly. Hence why we opted to use a
+# `stdenv.mkDerivation` instead.
+stdenv.mkDerivation rec {
+  pname = "MQTT-Explorer";
```

Comment:
See https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming
> The pname and the version attribute must not contain uppercase letters — e.g., "mplayer" instead of "MPlayer".

Reply:
Should I `s/MQTT-Explorer/mqtt-explorer/g`? The official full name of the app is `MQTT-Explorer` so I am not really entirely sure about this... or just the `pname`?

Reply:
Yes, I think both ones in lowercase would be better. It's quite common way to go in nixpkgs.